### PR TITLE
fix: previewScreen videostream bug

### PR DESCRIPTION
**Pull request by vivek9patel**

### CODE REVIEW

File: src/pages/PreviewScreen.jsx [hunk 1]
```diff
@@ -62,21 +46,37 @@ const PreviewScreen = () => {
...
-  return (
-    <div>
-      <div className="flex justify-center items-center">
-        <Preview
-          joinOnClick={join}
-          goBackOnClick={goBack}
-          messageOnClose={goBack}
-          onChange={onChange}
-        />
+  const isPreview = location.pathname.startsWith("/preview");
```
Comment (raviteja83) [inline src/pages/PreviewScreen.jsx:52]: why is this moved out of useEffect?

**Comment by eswarclynn** [comment]
Fixed by [PR#169](https://github.com/100mslive/100ms-web-v2/pull/169)
